codespell_lib/data/dictionary.txt

```diff
@@ -24535,31 +24639,58 @@ nexting->nesting, texting,
 niear->near
 niearest->nearest
 niether->neither
+nieve->naive
+nieveatay->naivete
+nievely->naively
+nife->knife
+nifes->knives
 nighbor->neighbor
 nighborhood->neighborhood
 nighboring->neighboring
 nighlties->nightlies
 nighlty->nightly
 nightfa;;->nightfall
+nighther->neither
 nightime->nighttime
+nihther->neither
+nimph->nymph
+nimphal->nymphal
+nimphean->nymphean
+nimphian->nymphean
+nimphlike->nymphlike
+nimpho->nympho
+nimphomania->nymphomania
+nimphomaniac->nymphomaniac
+nimphomaniacs->nymphomaniacs
+nimphos->nymphos
+nimphs->nymphs
+nimute->minute
 nimutes->minutes
 nin->inn, min, bin, nine,
 nineth->ninth
 ninima->minima
 ninimal->minimal
+ninimally->minimally
 ninimum->minimum
-ninjs->ninja
+ninj->ninja
+ninjs->ninja, ninjas,
 ninteenth->nineteenth
 ninties->nineties, 1990s,
 ninty->ninety, minty,
+nitch->niche
+nitched->niched
+nitches->niches
+nitching->niching
 nither->neither
+nives->knives, nieves, nines, dives, fives, hives, wives,
 nknown->unknown
 nkow->know
 nkwo->know
 nmae->name
 nmme->name
 nned->need
 nneeded->needed
+nnovisheate->novitiate
```

Comment:
Add plural?
`nnovisheates->novitiates`

And perhaps with a single `n`?
```
novisheate->novitiate
novisheates->novitiates
```

Reply:
Added.